ambari-logsearch/ambari-logsearch-server/src/main/resources/roles.json

```diff
@@ -0,0 +1,7 @@
+{
+  "roles": {
+    "user": [
+      "admin"
+    ]
+  }
```

Comment:
If you change this file structure like: 
```
  {
     "user1" : ["role1", "role2"],
     "user2" : ["role1", "role3"]
  }
```
than in the java code the user would be the key and the role list would be the value in the `simpleRolesMap` and the `List<...> RoleDao.getRolesForUser(username)` function implementation would be much simpler like 
```
simpleRolesMap.get(username)
```